pkg/preflight/assets/host-preflights.yaml

```diff
@@ -0,0 +1,66 @@
+# https://kurl.sh/docs/install-with-kurl/system-requirements
+apiVersion: troubleshoot.sh/v1beta2
+kind: HostPreflight
+metadata:
+  name: kurl-builtin
+spec:
+  collectors:
+    - cpu: {}
+    - memory: {}
+    - diskUsage:
+        collectorName: "Ephemeral Disk Usage"
+        path: /var/lib/kubelet
+    - tcpLoadBalancer:
+        collectorName: "Kubernetes API Server Load Balancer"
+        port: 6443
+        address: {{ .Installer.Spec.Kubernetes.LoadBalancerAddress }}
+        timeout: 3m
+        exclude: '{{ and .Installer.Spec.Kubernetes.LoadBalancerAddress .IsPrimary (not .IsJoin) (not .IsUpgrade) | not }}'
+  analyzers:
+    - cpu:
+        checkName: "Number of CPUs"
+        outcomes:
+          - fail:
+              when: "count < 4"
+              message: At least 4 CPU cores are required
+          - pass:
+              message: This server has at least 4 CPU cores
+    - memory:
+        checkName: "Amount of Memory"
+        outcomes:
+          - fail:
+              when: "< 8Gi"
+              message: At least 8Gi of memory is required
+          - pass:
+              message: The system has at least 8Gi of memory
+    - diskUsage:
+        checkName: "Ephemeral Disk Usage"
+        collectorName: "Ephemeral Disk Usage"
+        outcomes:
+          - warn:
+              when: "used/total > 70%"
+              message: /var/lib/kubelet is more than 70% full
+          - fail:
+              when: "total < 30Gi"
+              message: /var/lib/kubelet has less than 30Gi of total space
+          - pass:
+              message: /var/lib/kubelet has at least 30Gi disk space available
+    - tcpLoadBalancer:
+        checkName: "Kubernetes API Server Load Balancer"
+        collectorName: "Kubernetes API Server Load Balancer"
+        exclude: '{{ and .Installer.Spec.Kubernetes.LoadBalancerAddress .IsPrimary (not .IsJoin) (not .IsUpgrade) | not }}'
+        outcomes:
+          - warn: # fail
```

Comment:
What do these fail comments mean?

Reply:
i guess if i was confident in this check i would change those to fail